src/librustdoc/passes/collect_intra_doc_links.rs

```diff
@@ -1308,20 +1315,20 @@ impl LinkCollector<'_, '_> {
         let extra_fragment = &key.extra_fragment;
 
         match disambiguator.map(Disambiguator::ns) {
-            Some(ns @ (ValueNS | TypeNS)) => {
-                match self.resolve(path_str, ns, base_node, extra_fragment) {
+            Some(expected_ns @ (ValueNS | TypeNS)) => {
```

Comment:
I don't think renaming this variable makes things any more clear.

Reply:
I renamed it so that it's consistent with its usage as a field of `WrongNamespace` and because `ns` is a not very descriptive name. I do think this should have the same name as whatever we end up calling the field.

Reply:
> I do think this should have the same name as whatever we end up calling the field.

Fair enough, I agree with that.